core/src/test/java/com/epam/esm/service/validation/impl/InputDataValidatorImplTest.java

```diff
@@ -0,0 +1,36 @@
+package com.epam.esm.service.validation.impl;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import com.epam.esm.TestConfig;
+import com.epam.esm.exception.AbstractLocalizedCustomException;
+import com.epam.esm.exception.MismatchedIdValuesException;
+import com.epam.esm.service.validation.InputDataValidator;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+
+@ExtendWith({SpringExtension.class})
+@ContextConfiguration(classes = {TestConfig.class})
+class InputDataValidatorImplTest {
+
+	@Autowired
+	InputDataValidator dataValidator;
+
+	@Test
+	void pathAndBodyIdsCheckThrowsMismatchedIdValuesExceptionWhenIdsMismatch() {
```

Comment:
rename to 'pathAndBodyIdsCheckShouldThrowMismatchedIdValuesExceptionWhenIdsMismatch